### ISSUE TITLE
Required ARIA attribute is missing

### ISSUE THREAD

**Issue by jAigret-Bix** [uswds/uswds#4265]
Siteimprove: https://drive.google.com/file/d/1PnvCp1vXSM_DXur2Hzkl_1WcE6Pz5t50/view?usp=sharing

-[ ] Needs investigation
